### ISSUE TITLE
Rename My Portfolio to My Wallet

### ISSUE THREAD

**Issue by Xeldal** [bitshares/bitshares-ui#641]
What is the reason behind this recent change to "Portfolio"?  Is there confusion or a conflict in calling it My Wallet?  

To me the word portfolio indicates a specific book of assets you are tracking and trading and concerned with the overall profit and loss figures.  As a matter of accounting for a business, client, or personal investment;  This is a very specific use case.

Not everyone using the BitShares Client is there for trading, or tracking an investment.  Some use it purely as a wallet.  A place for there USD, EUR, BTC and BTS.  Maybe they only use it to load their visa card.  

Anyway, I understand the word Portfolio isn't limited to just this interpretation, I just think it is less fitting than the word Wallet.  And really I just don't like it, so I had to come up with more of a reason.  : )

I can imagine in the future you might be able to define various portfolios within your wallet.  Where each portfolio had a different user defined selection of coins from their wallet.  This might be used to track specific subsets of their total holdings.  Where they could make comparisons, adjustments etc.  Or maybe they only want to track the value of certain coins, and not include in the value all the various "junk" the account has picked up over the years.  ---- This would be a better use of the word Portfolio.

**Comment by wmbutler** [comment]
A wallet is a very generic term. For instance, I have a Local that contains 20 accounts. Each account has assets in it. Those assets make up my portfolio. I do not think it's a good idea to present a list of assets as a wallet.

**Comment by Xeldal** [comment]
It is generic, how is that a problem?

Is your reasoning that the term wallet would be referring to 2 different things?   
1. the list of accounts
2. the list of assets
Is that what you're saying?

While I don't necessarily think this is a conflict, it makes more sense to me to change the name of the list of accounts --if you must-- to something more appropriate.  If that's even necessary. (I don't think it needs a new name)  Everywhere else, you will find the term wallet referring to the balance of coins you posses and if multiple coins, referring to the list.     

It's very simple.  Each account has a wallet.  That is, the list of assets of that account.
The fact that you can bundle together a bunch of accounts does not change the fact that each account has a wallet.
The fact that you want then to call this bundle of wallets itself a wallet is also not a problem.  If it is, we should not be changing a word that effects everyone based on something that only effects a small portion of users.

For the majority of users, they will have 1 account, 1 wallet.  Or with your wording, they will not have a wallet, they'll have a portfolio.  That's strange.  (I bet they still call it a wallet though)

For the few that have multiple accounts bundled together. You're calling this a wallet, they only have 1  which contains many accounts.  But they can't view this one wallet's contents as one wallet like they're used to everywhere else, they must view it broken down into a bunch of "portfolios".  Again very strange.

And even with this change, you still have other instances of the word wallet.  What are you going to call a paper wallet for instance?  Can't call it a wallet right? that's a conflict with your model.  Can't call it a portfolio. Well, because that doesn't make any sense.

Naming it a portfolio is trying to solve a problem that doesn't exist.

**Comment by Xeldal** [comment]
"Wallet" refers to anything that contains coins, tokens, assets, etc. or anything that affects a transaction with these coins. How big or small or how many ways they are sliced and diced doesn't make any difference.  They are all wallets.  From a single paper wallet of one coin, to an account with multiple coins, to a bundle of accounts each with multiple coins.  They are all wallets.

In crypto these coins always come back to keys.  So a wallet has been essentially a list of keys.
As crypto has progressed these keys have taken on new abilities, new properties, new states. They carry a lot more than just control of coins.  

If anything I think the keys have out-grown the word wallet.  Using wallet to refer to the keys is a bit of a misapplication of terms.  Although this doesn't bother me.  It is still completely understandable.  A key is itself an asset so it still logically works with a bit of a stretch.  If this is the trend that the keys will continue to grow, it might be a mistake to continue to call the list of keys, a wallet.  

You however are moving the opposite direction.  Removing the word wallet from what it is and forcing it on something it already isn't.

BTW, this issue is definitely not a question.  Any questions I might have posed is meant to be rhetorical to insight a discussion.  I am proposing to change Portfolio to Wallet.

**Comment by tbone-bts** [comment]
I agree that "wallet" refers to too many different things and isn't a good choice to replace "My Portfolio".  If anything, "My Assets" or "My Holdings" would make more sense.  Actually, "My Assets"is not great because it sounds like it could refer to assets I've created.  "My Holdings" is better.  But I think "My Portfolio" is similar and probably equally sensible.

**Comment by btsfav** [comment]
as far as I know the general term used by Trading Platforms is Portfolio. I think it's fitting

**Comment by svk31** [comment]
@Xeldal we've had so many users confused over the terms account and wallet which is one of the reasons we didn't want to use the word wallet for this tab as well.

**Comment by landry314** [comment]
I take issue with the use of the word "My"; I think it sounds a little childish. Why use "My"? How does that differentiate or add anything? "Portfolio" by itself works fine.

A portfolio for an artist is a collection of the works the artist has made. A protfolio for a stock trader is the list of assets he currently holds (including short contracts).

Portfolio would be good on a trading platform but, I have to agree with Xeldal here, this is a GUI for more than just trading (it is also a simple wallet to hold keys and transfer funds) and as such, maybe portfolio sounds too specific to trading.

Why not go back to "Balances" or "Holdings". "Balances" sounds like the word a bank would use. "Portfolio" sounds like what a broker would use... I guess this is matter of what image we want to portray.

**Comment by wmbutler** [comment]
I actually added, "My" to increase the length a bit so the dollar value would fit under it without trailing off the left side. We have a lot of other things that are way more important than this.

- Wallets have Accounts.
- Accounts contain a Portfolio of Assets/Coins/Symbols
- Accounts will also soon contain a Watchlist

**Comment by Xeldal** [comment]
Landry314 makes some good points.  A standard portfolio would contain ALL your positions. Short, Margin, Open and otherwise.  As you have it now, the whole lot of tabs here is  really what should be called Portfolio(not suggesting this) If wallet is too confusing.  I think Balances is the most Appropriate.

- Wallets have Accounts
- Accounts have Balances
- Accounts have Watchlists

On the issue of "MY".  Adding to increase the text width is very poor justification.  If you use "MY" it should consistent and also be "MY Open Orders" , " MY Margin Positions"  and "My Watchlist"

I suggest dropping "MY" as well.  Just "Balances" will suffice.  

I have issues with the estimated values all over the place as well, but I'll save that for another issue.

**Comment by tbone-bts** [comment]
I agree with @landry314 and @Xeldal re: "Balances".  Also, although I don't think the word "My" is problematic in and of itself, it's inconsistent to use it only for one of the items.  Besides, logically, the names of these items should make sense prefaced by the word "Account", i.e. "Account Balances", "Account Activity", etc.  Not suggesting to actually preface each one with the word "Account though.

As for the word "Balances" not being long enough, thus leaving the estimated balance overhanging, I agree with @wmbutler that this is a detail that should be considered.  But I disagree with adding the word "My" to solve this problem for the reasons stated above.  I think the solution would be to remove the numbers after the decimal for any values over, say, 10,000.  This is an estimated number anyway, so it doesn't need to be especially precise.  

Speaking of the estimated Balance, I agree with @Xeldal that it doesn't need to be in multiple places.  In fact having it next to the menu items as it is with the new tables is confusing since it looks like a menu item.  But yeah, this should be discussed in another issue.

**Comment by landry314** [comment]
Yeah, so how about changing the ACCOUNT tab on he top to PORTFOLIO then "My Portfolio" to "Balances" then add a Watchlist section under PORTFOLIO? I am considering what @Xeldal said, "A standard portfolio would contain ALL your positions. Short, Margin, Open and otherwise." which is totally correct.

I feel like the word "account" is used differently in way too many places in this program. I was going to make an issue but never got around to it. I will soon. In summary, "account" is used:

----
ACCOUNT VALUE in top bar on right (change to ESTIMATED VALUE)
ACCOUNT in top bar (change to PORTFOLIO) 
"b" tab Accounts (change to "Followed Accounts")
Explore -> Accounts (this is fine)
Settings -> Accounts (change to "Manage Accounts")
ACCOUNT -> [CREATE A NEW ACCOUNT] (this is fine)
----

Also, "Estimated UNIT" in the upper left in the PORTFOLIO (ACCOUNT) is the same calculation as "ACCOUNT VALUE" on the upper right so why have it twice and why have it called different things? It is also called "Total Value" in the "b" menu to the left of ACCOUNT under "Accounts" (haha). It is quite an organizational mess, imho. The same calculation is called three different things. I feel like a naming coordination is in order here in many places. The program needs to feel clear where things are named well and the same throughout. Who agrees with me?

I know, @wmbutler, you feel there are more important things to work on and they are being worked on, but this is important too and doesn't require much to fix as renaming things is easy. We are working out the details right now. A consensus is all we need and then someone here could claim the issue as long as it fits into everyone's plans.

@Xeldal, the focus of this issue has changed, you might consider changing the name of the issue to make it more broad if you like... you have the power :-)

**Comment by tbone-bts** [comment]
I agree with @landry314 that it's confusing to have the estimated account value named differently in different places.  I also think it creates confusion and lack of clarity to have the estimated value contained in what looks like a menu item but isn't.  That needs to be more visually distinguished from the menu (or sub menu, in this case).     

As for the various references to "Account" listed by @landry314, I have zero problem with any of that since it all seems to refer to the same construct.  There is no inconsistency there, so I don't see any reason for confusion.  Therefore I would leave all references to "Account" as is, including in the main menu since the sub menu items on the Account page are specific to the selected/active **account**.  So changing "Account" to "Portfolio" in the main menu would just muddle things.

**Comment by Xeldal** [comment]
@wmbutler @svk31 The Question label for this issue is not appropriate.  This is a proposal not a question.

A lot of suggestions have come up here but i think for the most part they are being handled in other issues.  

Drop the "My" for consistency with the rest of the tables tabs
Change "Portfolio" to "Balances".  Portfolio is not applicable to just a table of balances.

The issue of the estimated balance being longer than the word Balances - if it's even an issue - can be solved in any number of ways.  Font size, letter spacing, display somewhere else, remove it entirely, or perhaps overhang is not an issue.  Anyway, that can be taken up in another issue.  

This is a simple rename.

**Comment by Xeldal** [comment]
Exodus knows how to do it. 
![](https://content.screencast.com/users/Enkianu/folders/Jing/media/ae65ec2f-ac42-46ef-895f-fbeacb4f9649/2017-11-08_1457.png)

Notice they have both a wallet(balances) and a portfolio.  The wallet is the list of coins and balances from which you can send and receive, while the portfolio is more of an analytical breakdown of allocations. 

If you want an example of how to do things, Exodus is great place to start.  Their wallet is beautiful and simple.  It immediately satisfies the average user who just wants to hold coins and doesn't over burden and over stimulate with all the complexities and exchange functions that launching a bitshares wallet does.  Be a wallet first and an exchange second.  Exodus also has an exchange which is just a simple shapeshift integration.  Bitshares should have this also.  Not necessarily through shapeshift but a simplified interal exchange that has the look and feel of shapeshift.  Most people will never be comfortable with the advanced trading layout and information.  Anyway this is going to be a separate issue.  

Back to "My Portfolio" vs "Balances"

It's a simple change.  Just do it.  Portfolio is something we can add later if we design more analytical tools into the wallet.  Right now the wallet is a bit of a mess.  I can't stand all the tables in tables in tables in boxs in boxs in more tables.  I often find myself searching around exclaiming "Where the fuck are the damn balances!!"  (true stroy)  Oh, they're hidden 3 levels deep under a non-sensical "portfolio" table button.   Then i switch to the Send tab just for some relief from the madness.

I'm not a fan of where the wallet is currently or with the direction the UI seems to be going but I'm waiting for the refactoring to settle a bit before I voice these criticisms.  It's not all bad, some of it looks really good.  I also don't know the greater vision you might have.

Be a wallet first, an exchange and all other specialized function second.

**Comment by landry314** [comment]
You are right, @Xeldal, I don't like how many layers there are and how I have to switch tabs for every little thing. Before, it was all on one page just in sections... now margin is hiding somewhere and tabs stay there so switching to accounts keeps the old state which I don't even remember and It is called My Porfolio which isn't good for someone just storing coins in a wallet and the "My" sounds like a two year old is keeping toys there... and the tab shouldn't be called ACCOUNT, there are too many things called that. It is the finance section, not account management.

Why not change ACCOUNT to ASSETS instead? That is more appropriate because it covers both people using it as a wallet and as a trading platform and it is where assets are managed, not where the account is managed.
